runtime/imported_values_memory_metering_test.go

```diff
@@ -427,3 +429,192 @@ func TestMemoryMeteringErrors(t *testing.T) {
 		assert.ErrorIs(t, err, testMemoryError{})
 	})
 }
+
+func TestImportedValueMemoryMeteringIterate(t *testing.T) {
```

Comment:
Oops that was a temporary name since the approach to test-writing was very different. It's iterating over an array of tests...

I'll give it a better name.